tests/ord-aggregator/tests/handler_test.go

```diff
@@ -605,8 +605,7 @@ func TestORDAggregator(stdT *testing.T) {
 		subscriptionProviderSubaccountID := testConfig.TestProviderSubaccountID
 		subscriptionConsumerSubaccountID := testConfig.TestConsumerSubaccountID
 		subscriptionConsumerTenantID := testConfig.TestConsumerTenantID
```

Comment:
With the new APIs seems the `TestConsumerTenantID` is not used for the subscription call but rather the "application name"? If that's the case, could we remove that property from the tests configurations and adapt the logs as well? And the the yaml configuration too(locally and productively because it's a little bit misleading to have all of the setup/configuration + the logs and not to use them.

https://github.com/kyma-incubator/compass/blob/saas-url/chart/compass/values.yaml#L967